2024-12-05 21:18:27,638 - src.logger - INFO - Starting App
2024-12-05 21:18:27,647 - apscheduler.scheduler - INFO - Scheduler started
2024-12-05 21:18:27,653 - root - ERROR - Unexpected error occurred while fetching indicators: 1 validation error for IndicatorSchema
default_settings
  Input should be a valid dictionary [type=dict_type, input_value=[['length', 'int', 14], [...], ['offset', 'int', 0]], input_type=list]
    For further information visit https://errors.pydantic.dev/2.10/v/dict_type
